CODEOWNERS

```
@@ -18,3 +18,8 @@ repository.yaml                         @HMRC/PlatUI
 *.scala                                 @HMRC/PlatUI
 *.sh                                    @HMRC/PlatUI
 *.xml                                   @HMRC/PlatUI
+.nvmrc                                  @HMRC/PlatUI
+.scalafmt.conf                          @HMRC/PlatUI
+.run_a11y_assessment_journey_tests.sh   @HMRC/PlatUI
+.run_acceptance_tests.sh                @HMRC/PlatUI
+.run_acceptance_tests_dev.sh            @HMRC/PlatUI
```

These should not have leading `.` ?
```suggestion
run_a11y_assessment_journey_tests.sh   @HMRC/PlatUI
run_acceptance_tests.sh                @HMRC/PlatUI
run_acceptance_tests_dev.sh            @HMRC/PlatUI
```

---

should this also include `docs/`, and maybe all the files immediately under `conf/` (though obviously not `conf/services/`) ?